client/src/App.tsx

```diff
@@ -19,7 +23,23 @@ const router = createBrowserRouter([
 ]);
 
 const App = () => {
-  return <RouterProvider router={router} />;
+  // return <RouterProvider router={router} />;
```

Comment:
remove the comment on the RouterProvider, so you turn it of when you add this comment on it.